eng/scripts/Language-Settings.ps1

```diff
@@ -0,0 +1,34 @@
+$Language = "android"
+
+function Get-AllPackageInfoFromRepo ($serviceDirectory)
+{
+  $allPackageProps = @()
+  Push-Location $RepoRoot
+
+  if ($serviceDirectory)
+  {
+    $properties = gradle ":sdk:$serviceDirectory:retrieveProjectProperties" -q
+  }
+  else
+  {
+    $properties = gradle retrieveProjectProperties -q
+  }
+  Pop-Location
+  foreach($line in $properties)
+  {
+    if ($line -match "^(?<version>.*)?~(?<name>.*)?~(?<group>.*)~(?<projDir>.*)")
```

Comment:
Is there a reason you use `~` as a delimiter? If not could you do what we did for .NET, see https://github.com/Azure/azure-sdk-for-net/pull/22352/ and just put quotes and use space delimiter.

Reply:
No reason for usign `~` except that I expect it will never occur within string. I can't use space because am also retrieving the project directory which may contain space on some peoples PCs